microsoft/utils/setup_AITL.py

```diff
@@ -0,0 +1,299 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+# This script is to create/update a custom role and grant to Service Principal for Partner in Production Env
+# Debug sample: .\Setup_AITL.py -r "AzureLSGDelegator_test" -p "AzureLinuxTestingService" -s "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+# User sample:  .\Setup_AITL.py
+
+import argparse
+import json
+import subprocess
+import sys
+import time
+
+
+def call_cmd(command: str) -> subprocess.CompletedProcess[str]:
+    # Run the command and capture the return code
+    try:
+        result = subprocess.run(
+            command,
+            shell=True,
+            check=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            text=True,
+        )
+        return_code = result.returncode
+        print(f"Return Code: {return_code} for cmd: {command}")
+        # output = result.stdout
+        # print(f"Output:\n{output}") # can open for debug purpose
+        return result
+    except subprocess.CalledProcessError as e:
+        # If the command returns a non-zero exit code, you can handle the error here
+        print(f"Command failed with error: {e.stderr}")
```

Comment:
It doesn't need to catch error, if it just exists. Let the exception throw out directly.